inc/functions_modcp.php

```diff
@@ -145,6 +145,22 @@ function send_report($report, $report_type='post')
 {
 	global $db, $lang, $forum, $mybb, $post, $thread, $reputation, $user;
 
+	$report_reason = '';
+	if($report['reasonid'])
+	{
+		$query = $db->simple_select("reportreasons", "title", "rid = '{$report['reasonid']}'", array('limit' => 1));
```

Comment:
If Jordan is not available we could just push this and then do another commit to cast it as int.